frontend/src/components/common/Verortung.vue

```diff
@@ -188,16 +312,29 @@ export default class Verortung extends Mixins(GeodataEaiApiRequestMixin, SaveLea
     );
   }
 
+  get kitaplanungsbereiche(): Array<KitaplanungsbereichDto> {
+    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), ["nummer"]);
```

Comment:
Das mit `nummer` wird hier nicht funktionieren, da es das Attribute in Kitaplanungsbereich nicht gibt. Den Fehler hatte ich in einem vorherigen Commit eigentlich schon behoben gehabt

```suggestion
    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), (k) => Number(k.kitaPlbT));
```

Reply:
Das Attribut gibt nun. Es wurde in der GeodatenEAI nicht gemappt. Nun wird es mitgeliefert.

Reply:
Response Body Bauvorhaben mit Verortung.

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/8cab299d-8b5e-4471-99e6-d786b6d9d2d4)

Reply:
Berichtigt und anstelle "nummer" wird "kitaPlb" verwendet.